meson_options.txt

```diff
@@ -51,3 +51,4 @@ option('efi_sbat_distro_summary', type : 'string', value : '', description : 'SB
 option('efi_sbat_distro_pkgname', type : 'string', value : '', description : 'SBAT distribution package name, e.g. fwupd')
 option('efi_sbat_distro_version', type : 'string', value : '', description : 'SBAT distribution version, e.g. fwupd-1.5.6.fc33')
 option('efi_sbat_distro_url', type : 'string', value : '', description : 'SBAT distribution URL, e.g. https://src.fedoraproject.org/rpms/fwupd')
+option('efi_binary', type: 'boolean', value: 'true', description : 'build included UEFI binary')
```

Comment:
maybe `plugin_uefi_capsule_efi_binary`? It might not be the only EFI binary we every use, vis-a-vis superio. Same for the description.

Reply:
In 1_5_X?  I would think such a thing wouldn't backport that far.

Reply:
Hmm, maybe not. Maybe one big switch for "all the efi stuff" is okay